src/core/qgsuserprofilemanager.h

```diff
@@ -44,6 +44,18 @@ class CORE_EXPORT QgsUserProfileManager : public QObject
 
   public:
 
+    /**
+     * User profile selection policy.
+     *
+     * \since QGIS 3.32
+     */
+    enum UserProfileSelectionPolicy
```

Comment:
can you move this to the Qgis class and promote to a enum class? (also add the Q_ENUM macro accordingly)

Reply:
Sure